nixos/modules/services/networking/xrdp.nix

```diff
@@ -180,6 +196,8 @@ in
     users.groups.xrdp = {};
 
     security.pam.services.xrdp-sesman = { allowNullPassword = true; startSession = true; };
+
+    warnings = lib.optional (!config.hardware.pulseaudio.enable) "xrdp: pulseaudio is not enabled, sessions will have no audio";
```

Comment:
I am personally not interested in sound via xrdp and don't have pulseaudio enabled on my system (I use pipewire). This warning therefore seems too eager to me.

Reply:
Do you think it would be better to add a extra option to hide this warning?

Reply:
Considering that the user may use a PulseAudio emulation layer (like `pipewire-pulse`), having this warning looks like a bad idea.

Reply:
Do pipewire-pulse support pulseaudio modules?

One thing is client compatibility, another one is plugin compatibility.

Reply:
This warning is still a bad idea, generally warnings are used to inform the user about broken configs but having `pulseaudio` module disabled is not broken.

This warning could make sense if the user has `pulseaudio` disabled and this had some flag like `enableAudio`.

Actually having this option looks like a good idea anyway, so I think we should include this flag.

Reply:
I added it and default to true if pulseaudio is enabled